### PR TITLE
Add support for XVG address

### PR DESCRIPTION
**Pull request by kleo2012**
Added support for XVG address format

## Issue number
#140

## List of features added/changed

Add XVG on:
src/index.ts
src/tests/index.test.ts
README.md

Already sorted by Cointype on Typescript files and alphabetically on README.md

NOTE: If you are adding new coin address support, please make sure to add a reference link to README so that reviewer can verify.

## How Has This Been Tested?
Local test run
TravisCI

## Screenshots (if appropriate):
LocalTest

## Checklist:

- [x] My code follows the code style of this project.
- [x] My code implements all the required features.

### CODE REVIEW

**Comment by Arachnid** [comment]
I have updated the Travis build to check that entries in the formats, tests, and README lists are in the correct order. Please check your PR and update it to maintain correct ordering in these lists.

**Comment by kleo2012** [comment]
Done.

File: README.md [hunk 1]
```diff
@@ -64,7 +64,9 @@ This library currently supports the following cryptocurrencies and address forma
  - XLM (ed25519 public key)
  - XRP (base58check-ripple)
  - XTZ (base58check)
+ - XVG (base58check)
```
Comment (makoto) [inline README.md:67]: Can you provide reference link specifying that XVG is base58check ?

File: src/__tests__/index.test.ts [hunk 2]
```diff
@@ -114,6 +114,13 @@ const vectors: Array<TestVector> = [
       { text: 'cxa4524257b3511fb9574009785c1f1e73cf4097e7', hex: '01a4524257b3511fb9574009785c1f1e73cf4097e7' },
     ],
   },
+  {
+    name: 'XVG',
+    coinType: 77,
+    passingVectors: [
+      { text: 'D7MKQnLxXEqn84PN42jWAVhvrXEuULLV9r', hex: '76a914183ffcc41f3095bea7ff324e52a65b46c74126e188ac' },
```
Comment (makoto) [inline src/__tests__/index.test.ts:121]: Where did you find this test data? Please specify source (eg: test code, block explorer).

File: src/index.ts [hunk 3]
```diff
@@ -623,6 +623,7 @@ const formats: IFormat[] = [
   hexChecksumChain('ETH', 60),
   hexChecksumChain('ETC', 61),
   getConfig('ICX', 74, icxAddressEncoder, icxAddressDecoder),
+  bitcoinBase58Chain('XVG',77, [[0x1E]], [[0x21]]),
```
Comment (makoto) [inline src/index.ts:626]: Can you specify where you found the version number of the [0x1E]], [[0x21]?

File: src/__tests__/index.test.ts (hunk 2 above)
Comment (kleo2012) [reply src/__tests__/index.test.ts:121]: in JSON section 
https://vergeexplorer.com/tx/881facd64fc4a2d56b3b76598d45ff51abcc29028220281fafa9247a46a7ecb0
verge

File: src/index.ts (hunk 3 above)
Comment (makoto) [reply src/index.ts:626]: Found it at https://pkg.go.dev/github.com/assetsadapterstore/verge-adapter/verge#pkg-variables

**Comment by kleo2012** [comment]
and there will be no payments?

**Pull request merged**